src/main/kotlin/graphics/scenery/Arrow.kt

```diff
@@ -125,10 +124,10 @@ class Arrow(var vector: GLVector) : Node("Arrow"), HasGeometry {
 
     /**
      * Adds a line point to the line.
-	  * TODO: make this method private
      *
      * @param p     The vector containing the vertex data
      */
+	 private
```

Comment:
Could you please put the visibility modifiers on the same line as the function definition? Also, indentation seems to be a bit off

Reply:
Done.